README.md

```diff
@@ -1,2 +1,25 @@
-# hw01
-Repository for homework 01
+---
+output:
```

Comment:
I am confused about the "output" section. Perhaps you can delete the three lines, from `output` to `html_document: default`.